aws_google_auth/__init__.py

```diff
@@ -66,7 +71,7 @@ def cli(cli_args):
         config = resolve_config(args)
         process_auth(args, config)
     except google.ExpectedGoogleException as ex:
-        print(ex)
+        logging.exception(ex)
```

Comment:
section should be:
```
    except google.ExpectedGoogleException as ex:
        print(ex)
        sys.exit(1)
    except KeyboardInterrupt:
        pass
    except Exception as ex:
        logging.exception(ex)
```
The ExpectedGoogleException is used for application flow, and so should eat the exception trace - see scenarios where user supplies invalid password.

Reply:
Thanks for the explanation @stevemac007 ! I appreciate it.